diff -uraN cosmic-session-epoch-1.0.0-alpha.5/Justfile omv-cosmic-session-epoch-1.0.0-alpha.5/Justfile
--- cosmic-session-epoch-1.0.0-alpha.5/Justfile	2025-01-02 17:34:01.000000000 +0100
+++ omv-cosmic-session-epoch-1.0.0-alpha.5/Justfile	2025-01-10 18:16:45.454866751 +0100
@@ -13,14 +13,14 @@
 orca := '/usr/bin/orca'
 cosmic_dconf_profile := '/usr/share/dconf/profile/cosmic'
 
-bindir := prefix + '/bin'
-systemddir := prefix + '/lib/systemd/user'
-sessiondir := prefix + '/share/wayland-sessions'
-applicationdir := prefix + '/share/applications'
+bindir := rootdir + prefix + '/bin'
+systemddir := rootdir + prefix + '/lib/systemd/user'
+sessiondir := rootdir + prefix + '/share/wayland-sessions'
+applicationdir := rootdir + prefix + '/share/applications'
 
-all: _extract_vendor build
+all: _extract_vendor build-release
 
-build:
+build-release:
         XDP_COSMIC={{xdp_cosmic}} ORCA={{orca}} cargo build {{cargo_args}}
 
 # Installs files into the system
